src/scripts/itinerary_scripts.js

```diff
@@ -0,0 +1,20 @@
+var season = itinerary.recommendedSeason;
+
+function changeSeason() {
+  switch (season) {
+    case 'WINTER':
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7181090)
```suggestion
    case "WINTER":
```